src/screens/02-Clock/JsAndCssClock.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useRef, useState } from 'react';
+import styles from './style.module.css';
+import cx from 'classnames';
+
+export const JsAndCssClock = () => {
+	const [secondsDegrees, setSecondsDegrees] = useState(0);
+	const [minsDegrees, setMinsDegrees] = useState(0);
+	const [hourDegrees, setHourDegrees] = useState(0);
+	const [isReady, setIsReady] = useState(false);
+
```

Comment:
maybe better will be to create just one useState:

`[time, setTime] = useState({seconds: 0, minutes: 0, hours: 0, isReady: false})`